src/Microsoft.AspNet.Razor/Runtime/TagHelpers/TagMode.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.Razor.Runtime.TagHelpers
+{
+    /// <summary>
+    /// The mode in which a tag should render.
```

Comment:
Like the `TagStructure` comment changes. My comment applied here as well e.g. `The mode in which an element should render.` or `The tags included when rendering an element.` here and `Include both start and end tags.` for `StartTagAndEndTag`.